src/server/pfs/fuse/fuse.go

```diff
@@ -73,6 +74,11 @@ func Mount(c *client.APIClient, target string, opts *Options) (retErr error) {
 			}
 		}
 	}()
+	fmt.Println("Uploading files to Pachyderm...")
+	// Rendering progress bars for thousands of files significantly slows down
+	// throughput. Disabling progress bars takes throughput from 1MB/sec to
+	// 200MB/sec on my system, when uploading 18K small files.
+	progress.Disable()
```

Comment:
Instead of hard disabling, maybe control it via options?

Reply:
I don't think we should ever give people the option to turn on something which destroys performance so badly 😄 especially not with `pachctl mount` which - as it's an interactive mount - is potentially more likely to deal with many small files.